benchmarks/extract_dataset_configs/imagenet.py

```diff
@@ -23,4 +23,5 @@
             dict(type='CenterCrop', size=224),
             dict(type='ToTensor'),
             dict(type='Normalize', **img_norm_cfg),
+            dict(type='Collect', keys=['img', 'gt_labels'])
```

Comment:
has nothing to do with this pr, fix previous bug, please refer to: https://github.com/alibaba/EasyCV/pull/6